crates/kitsune_p2p/fetch/CHANGELOG.md

```diff
@@ -7,8 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
-- Fix an issue with merging fetch contexts where merging an item with a context with an item that did not could result
-  in the removal of the context.
+## 0.3.0-beta-dev.7
+
+- Fix an issue with merging fetch contexts where merging an item with a context with an item that did not could result in the removal of the context.
```

Comment:
formatting, `could result` is not indented any more. Not sure if this affects display

Reply:
looks like it removed a line-break. seems fine to me.